### PR TITLE
Add documentation for Flick Electric sensor

### PR DESCRIPTION
**Pull request by ZephireNZ**
## Proposed change

Adds documentation for the Flick Electric integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/home-assistant#30696
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_integrations/flick_electric.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+This integration can be configured via the Home Assistant frontend.
+
+- Go to **Configuration** -> **Integrations**.
+- Click on the `+` in the bottom right corner to add a new integration.
+- Search and select the **Flick Electric** integration form the list.
+- Follow the instruction on screen to add the sensor.
+
+<div class='note'>
+The configuration uses the client ID and secret used by the app at time of release. If this stops working, you can find the new ones by using a MITM proxy with the mobile app. The app will call `https://api.flick.energy/identity/oauth/token` with the `client_id` and `client_secret`.
```
Comment (ZephireNZ) [inline source/_integrations/flick_electric.markdown]: The inline code in this does not show up correctly in the preview build, but I'm using the same format as [the Sonos page](https://github.com/home-assistant/home-assistant.io/blob/25b3e7c7fb36d534ab0fb5d6b3e2450eef57d1de/source/_integrations/sonos.markdown#service-sonossnapshot) so not sure what I'm doing wrong.

Comment (frenck) [reply source/_integrations/flick_electric.markdown]: Add enters around markdown. The parser needs to be able to switch context between HTML & Markdown. A new line will do that.

Fixed it 😉

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @ZephireNZ! 👍

**Pull request merged**